share/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@
 # terms of the BSD-3 license. We welcome feedback and contributions, see file
 # CONTRIBUTING.md for details.
 
+if(NOT WIN32)
```

Comment:
Does this mean no GLVis icon on Windows?

Reply:
At least for now; I'll need to find a substitute for `xxd` on Windows, or generate an `glvis-icon.ico` file to include in the Windows executable directly.